plugins/woocommerce/src/Internal/Admin/Events.php

```diff
@@ -250,4 +253,21 @@ protected function is_merchant_email_notifications_enabled() {
 		// All checks have passed.
 		return true;
 	}
+
+	/**
+	 *   Refresh transient for the following DataSourcePollers on wc_admin_daily cron job.
+	 *   - PaymentGatewaySuggestionsDataSourcePoller
+	 *   - RemoteFreeExtensionsDataSourcePoller
+	 */
+	protected function possibly_refresh_data_source_pollers() {
+		$completed_tasks = get_option( 'woocommerce_task_list_tracked_completed_tasks' );
+
+		if ( ! in_array( 'payments', $completed_tasks, true ) ) {
```

Comment:
Good thought on adding task checks here! Should we also add `woocommerce-payments` here? Since if the task overrides it, I believe `payments` won't be completed.

Reply:
Please proceed to merge on another approval if this is resolved/found irrelevant

Reply:
> Good thought on adding task checks here!

+1 👍